### ISSUE TITLE
Create web form for "Resources"

### ISSUE THREAD

**Issue by bnvk** [opensourcedesign/opensourcedesign.github.io#82]
As per the goal of making it easy for designers who are not comfortable using GitHub to participate in OSD, we should make a nice web form to "Add Design Resource" without having to create a Github account. Ideally, we should have a dropdown menu which sorts resources by category. Currently, the following categories exist:

- Tools
- Material / Content
- Images
- Inspiration
- Icons
- Fonts
- CSS Frameworks
- General Design Reads
- Open Source and Design Reads
- Searching The Creative Commons
- Learn to Code
- Learn to Design
- Other nice stuff

The later half of those could be clarified / condensed a bit perhaps.

**Comment by ei8fdb** [comment]
Hmm, theres a distinct lack of user research methods in that list! I'd suggest adding the following to the list:

- recruitment of research participants
- contextual interviews
- formative usability testing
- card sorting
- guerrilla usability testing
- task analysis
- participant interviews
- measuring analytics and metrics
- heuristic review
- creation of personas
- creating surveys
- lab based usability testing
- prototyping (paper, wireframes, interactive..etc)
- summative usability testing
- using System Usability Scale (SUS)

**Comment by A-GHOSH-dev** [comment]
Can I contribute here?

**Comment by jdittrich** [comment]
@A-GHOSH-dev yes! I don't know what the issue will focus on, it seems that it is currently both about how the form should be structured as well as about implemeting it at some point ( @bnvk, @ei8fdb – do we have a technology we usually use for form submits like this?)

**Comment by A-GHOSH-dev** [comment]
> @A-GHOSH-dev yes! I don't know what the issue will focus on, it seems that it is currently both about how the form should be structured as well as about implemeting it at some point ( @bnvk, @ei8fdb – do we have a technology we usually use for form submits like this?)

Okay..... So just tell me what exactly the issue is. As far as I understood, It is I have to add a dropdown which user will select and as per which the content of the webpage will be filtered.

**Comment by A-GHOSH-dev** [comment]
Just guide me a bit. This will be my first Open Source Contribution. 

> > @A-GHOSH-dev yes! I don't know what the issue will focus on, it seems that it is currently both about how the form should be structured as well as about implemeting it at some point ( @bnvk, @ei8fdb – do we have a technology we usually use for form submits like this?)
> 
> Okay..... So just tell me what exactly the issue is. As far as I understood, It is I have to add a dropdown which user will select and as per which the content of the webpage will be filtered.

Just Guide me a bit, This will be my first open source contribution.

**Comment by jdittrich** [comment]
Closed upon "Close decision by end of 2025"-Tag
